docs/modules/ROOT/pages/glossary.adoc

```diff
@@ -47,3 +47,41 @@ can be referenced in the tests as `/data/request.json`.
 
 The name of a resource is independent of the used operating system;
 in particular, the path separator is always a slash `/`.
+
+== Table Transformer
```

Comment:
I think it makes sense to put this reference somewhere here https://jbehave.org/reference/stable/tabular-parameters.html